stage0/src/main.rs

```diff
@@ -259,6 +259,23 @@ pub extern "C" fn rust64_start(encrypted: u64) -> ! {
         zero_page.hdr.setup_data = &setup_data.header as *const oak_linux_boot_params::SetupData;
     }
 
+    if let Ok(cmdline_size) = fwcfg.read_cmdline_size() {
+        if cmdline_size > 0 {
+            let mut buf = alloc
+                .alloc(Layout::from_size_align(cmdline_size as usize, 1).unwrap())
```

Comment:
Do we want to change this `unwrap` to expect too?

Reply:
I don't think this can actually fail, because (a) cmdline_size is an `u32` and therefore by always smaller than `isize::MAX`, and we fix the alignment itself to be 2^0 = 1. But I've changed it to an `expect()`, just in case.